spec.html

```diff
@@ -20773,13 +20908,13 @@
 
       <!-- es6num="15.2.1.19" -->
       <emu-clause id="sec-toplevelmoduleevaluationjob" aoid="TopLevelModuleEvaluationJob">
-        <h1>Runtime Semantics: TopLevelModuleEvaluationJob ( _sourceText_)</h1>
-        <p>A TopLevelModuleEvaluationJob with parameter _sourceText_ is a job that parses, validates, and evaluates _sourceText_ as a |Module|.</p>
+        <h1>Runtime Semantics: TopLevelModuleEvaluationJob ( _sourceText_, _hostDefined_ )</h1>
+        <p>A TopLevelModuleEvaluationJob with parameters _sourceText_ and _hostDefined_ is a job that parses, validates, and evaluates _sourceText_ as a |Module|.</p>
         <emu-alg>
           1. Assert: _sourceText_ is an ECMAScript source text (see clause <emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>).
           1. Let _realm_ be the running execution context's Realm.
-          1. Let _m_ be ParseModule(_sourceText_).
-          1. If _m_ is an abrupt completion or any other implementation defined error indication, then
+          1. Let _m_ be ParseModule(_sourceText_, _hostDefined_).
+          1. If _m_ is an abrupt completion, then
```

Comment:
ScriptEvaluationJob currently uses "[...] an indication of one or more parsing errors and/or early errors", so instead of an *implementation defined error indication*, ParseModule could return a List of errors. (Plus additional notes that the ordering and number of errors in that List is implementation dependent.)